torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -893,7 +909,18 @@ def crop_bounding_box(
 
 
 def crop_mask(mask: torch.Tensor, top: int, left: int, height: int, width: int) -> torch.Tensor:
-    return crop_image_tensor(mask, top, left, height, width)
+    if mask.ndim < 3:
```

Comment:
We need this handling here, because `_pad_with_scalar_fill` doesn't support 2d images, right? Otherwise, I don't see anything in `crop_image_tensor` that would require 3 or more dims.

Reply:
Correct. The issue come from pad:
```
  File "./vision/torchvision/prototype/transforms/functional/_geometry.py", line 890, in _pad_with_scalar_fill
    num_channels, height, width = shape[-3:]
```